src/main/java/com/aetherteam/aether_genesis/data/providers/GenesisBlockStateProvider.java

```diff
@@ -109,4 +111,59 @@ private void logWallSidePart(MultiPartBlockStateBuilder builder, ModelFile model
     public void woodWallBlock(WallBlock block, Block baseBlock, String location, String modid) {
         this.wallBlockInternal(block, this.name(block), new ResourceLocation(modid, "block/" + location + this.name(baseBlock)));
     }
+
+    public void skyrootCraftingTable(Block block) {
+        ModelFile workbench = this.models().cube(this.name(block),
+                new ResourceLocation(Aether.MODID, "block/construction/skyroot_planks"),
+                this.extend(this.texture(this.name(block), "utility/"), "_top"),
+                this.extend(this.texture(this.name(block), "utility/"), "_front"),
+                this.extend(this.texture(this.name(block), "utility/"), "_side"),
+                this.extend(this.texture(this.name(block), "utility/"), "_front"),
+                this.extend(this.texture(this.name(block), "utility/"), "_side"));
+        this.getVariantBuilder(block).partialState().addModels(new ConfiguredModel(workbench));
+    }
+
+    public void furnace(Block block) {
+        String blockName = this.name(block);
+        ResourceLocation side = this.extend(this.texture(this.name(block), "utility/"), "_side");
+        ResourceLocation front_on =  this.extend(this.texture(this.name(block), "utility/"), "_front_on");
+        ResourceLocation front =  this.extend(this.texture(this.name(block), "utility/"), "_front");
+        ResourceLocation top = this.extend(this.texture(this.name(block), "utility/"), "_top");
+        ModelFile normal = this.models().orientable(blockName, side, front, top);
+        ModelFile lit = this.models().orientable(blockName + "_on", side, front_on, top);
+        this.getVariantBuilder(block).forAllStatesExcept((state) -> {
+            Direction direction = state.getValue(AbstractFurnaceBlock.FACING);
+            if(state.getValue(AbstractFurnaceBlock.LIT))
```

Comment:
Another nitpick, sorry, but have a space between the `if` and `(`

Reply:
got it, will fix it

Reply:
fix